examples/01-hello-world.mts

```diff
@@ -9,50 +8,52 @@
  *   DEBUG=json-rules-engine node ./examples/01-hello-world.js
  */
 
-require('colors')
-const { Engine } = require('json-rules-engine')
+import "colors";
```

Comment:
Tell me more about the change to `.mts` decision specifically for `examples` - the idea of the examples is to cater to the lowest common denominator. Seems like we're narrowing the audience by moving away from .js?

Reply:
I guess this is more of a question of what the "lowest common denominator" is in 2024?
For instance should the examples be common js with `require` statements or can we safely move to module syntax.  Node's had support for modules for some time now and with a move to drop the EOL versions of node I think using the module syntax is probably ok.
I think for most people Typescript is the default choice in this day-and-age so it's probably worth having some examples in Typescript. I could see an argument made for actually having both.

Reply:
@chris-pardy ty, makes sense. I no longer work in this ecosystem, so I'll defer to your judgement on this.